src/RuleRight.php

```diff
@@ -411,6 +411,7 @@ public function showForm($ID, array $options = [])
             'match_operators' => $this->getRulesMatch(),
             'conditions' => static::getConditionsArray(),
             'rand' => mt_rand(),
+            'short' => true,
```

Comment:
Should only be the "short" form in the context of the Rules tab in the Entity form I think.